client/my-sites/sidebar/sidebar.jsx

```diff
@@ -738,6 +750,7 @@ function mapStateToProps( state ) {
 		isManageSectionOpen,
 		isAtomicSite: !! isSiteAutomatedTransfer( state, selectedSiteId ),
 		isVip: isVipSite( state, selectedSiteId ),
+		isFullSiteEditing: isSiteUsingFullSiteEditing( state, siteId ),
```

Comment:
Ditto on the suggested naming like `showCustomizerLink: ! isSiteUsingFullSiteEditing( state, siteId ),`